.ci/scripts/generate-portainer-manifests-from-chart.sh

```diff
@@ -15,7 +15,7 @@
 # 2. Remove the header produced by helf --dry-run
 # 3. Remove references to helm in rendered manifests (no point attaching a label like "app.kubernetes.io/managed-by: Helm" if we are not!)
 
-helm install --no-hooks --namespace zorgburger --set disableTest=true --dry-run zorgburger charts/portainer \
+helm install --no-hooks --namespace zorgburger --set service.type=NodePort --set disableTest=true --dry-run zorgburger charts/portainer \
```

Comment:
What is this change about?

Reply:
The script runs the helm templating process twice - the first one generates nodeport config (but this was previously implicit), and the second generates loadbalancer config. This change simply make nodeport on the first run more explicit, for readability.